smithy-aws-protocol-tests/model/awsQuery/main.smithy

```diff
@@ -1,5 +1,15 @@
 $version: "2.0"
 
+metadata suppressions = [
```

Comment:
Is it prohibitive to use the `@suppress` trait instead of a metadata suppression? It's a nice bit of documentation on that specific member as well that it's intending to test the behavior.